packages/webcomponents/src/components/business-forms/owner-form/owner-form.tsx

```diff
@@ -299,7 +299,7 @@ export class BusinessOwnerForm {
                   class={`btn btn-primary jfi-submit-button${this.isLoading ? ' jfi-submit-button-loading' : ''}`}
                   onClick={() => this.handleAddOwner()}
                   disabled={this.isLoading}>
-                  {this.isLoading ? LoadingSpinner() : this.submitButtonText}
+                  {this.isLoading ? ButtonSpinner() : this.submitButtonText}
```

Comment:
I see you added in the ButtonSpinner here, but do you want to use the new Button component you made as well? Then you dont't need to import the ButtonSpinner and can just pass this.isLoading to it?

Reply:
Yes, we should do that eventually but it felt like a slippery slope - I think we can have a PR that just replaces all buttons with the new one. We will have to expand on the Button component to handle other styles of buttons such as `btn-danger`